### PR TITLE
improved sizing of viewport and tiles

### PR DESCRIPTION
**Pull request by brewster1134**

### CODE REVIEW

**Comment by btburton42** [comment]
good enhancements! :+1:

File: demo/tiler_demo.css [hunk 1]
```diff
@@ -5,25 +5,26 @@ html, body {
...
     position: absolute;
-    top: 0;
+    z-index: 3;
     right: 0;
-    z-index: 3; }
+    top: 0; }
   .tiler-viewport .tiler-tile {
-    background-position: center; }
+    background-position: center;
+    transition-duration: 0.25s !important; }
```
Comment (btburton42) [inline demo/tiler_demo.css]: is it good to be forcing transition timing here?

Comment (btburton42) [inline demo/tiler_demo.css]: whoops. that's demo. derp.

File: lib/tiler.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .tiler-viewport {
-  overflow: hidden;
-  position: relative; }
+  position: relative;
+  background: white;
```
Comment (btburton42) [inline lib/tiler.css]: how is this added to the lib and not in the `src` changes?

Comment (brewster1134) [inline lib/tiler.css]: i think these particular properties were already in the checked in src, just not the compiled lib files.... sloppy incomplete release basically...

**Comment by btburton42** [comment]
No this was every tile on the site. One line css change to cq-ui and we were back in business.

**Comment by brewster1134** [comment]
ah ok i understand now... so this new approach with tiler is legit, it just needed some style changes in response... nice.

**Pull request merged**